client/src/components/shared/Info/Info.js

```diff
@@ -0,0 +1,332 @@
+                    <ExternalLink
+                      isMapView={false}
+                      url={icoUrl(findata.ico)}
+                      text={showNumberCurrency(zisk)}
+                    />
+                  </span>
+                  {findata.zisk_trend !== 0 && <Trend trend={8} />}
+                </td>
+              </tr>
+            )}
+            {findata.ico &&
+              trzby !== undefined &&
+              trzby !== 0 && (
+              <tr>
+                <td colSpan="2">
+                  <span>
+                    <strong>Tržby v 2016:</strong>&nbsp;
+                    <ExternalLink
+                      isMapView={false}
+                      url={icoUrl(findata.ico)}
+                      text={showNumberCurrency(trzby)}
+                    />
+                  </span>
+                  {findata.trzby_trend !== 0 && <Trend trend={10} />}
+                </td>
+              </tr>
+            )}
+            {data.total_contracts !== null &&
+              data.total_contracts > 0 && (
+              <tr>
+                <td colSpan="2">
+                  <strong>Verejné zákazky:</strong>&nbsp;
+                  <ExternalLink
+                    isMapView={false}
+                    url={`http://www.otvorenezmluvy.sk/documents/search?utf8=%E2%9C%93&q=${
+                      entity.entity_name
+                    }`}
+                    text={showNumberCurrency(data.total_contracts)}
+                  />
+                </td>
+              </tr>
+            )}
+            {data.advokati_data.length >= 1 && (
+              <tr>
+                <td colSpan="2">
+                  <ExternalLink
+                    isMapView={false}
+                    url={`http://datanest.fair-play.sk/searches/quick?query_string=${
+                      entity.entity_name
+                    }`}
+                    text={'Advokát'}
+                  />
+                </td>
+              </tr>
+            )}
+            {data.nadacie_data.length >= 1 && (
+              <tr>
+                <td colSpan="2">
+                  <ExternalLink
+                    isMapView={false}
+                    url={`http://datanest.fair-play.sk/searches/quick?query_string=${
+                      entity.entity_name
+                    }`}
+                    text={'Nadácia'}
+                  />
+                </td>
+              </tr>
+            )}
+            {data.auditori_data.length >= 1 && (
+              <tr>
+                <td colSpan="2">
+                  <ExternalLink
+                    isMapView={false}
+                    url={`http://datanest.fair-play.sk/searches/quick?query_string=${
+                      entity.entity_name
+                    }`}
+                    text={'Auditor'}
+                  />
+                </td>
+              </tr>
+            )}
+            {data.sponzori_stran_data.length >= 1 && (
```

Comment:
Why >= 1? Is it a big problem if we render it as empty?

Reply:
It is not a problem, but api can return a lot of different data (etc. 10 arrays) and a lot of companies has many of them empty. So I think it is better ignore empty arrays insted of render something like (Contracts:  0 \n Trends: 0 \n ...)